third_party/lua/luadebug.lua

```diff
@@ -807,7 +997,9 @@ if color_maybe_supported and not os.getenv("NO_COLOR") then
     COLOR_BLUE = string.char(27) .. "[94m"
     COLOR_YELLOW = string.char(27) .. "[33m"
     COLOR_RESET = string.char(27) .. "[0m"
-    GREEN_CARET = string.char(27) .. "[92m => " .. COLOR_RESET
+    CARET = string.char(27) .. "[92m => " .. COLOR_RESET
+    CARET_SYM = string.char(27) .. "[92m=>" .. COLOR_RESET
+    BREAK_SYM = string.char(27) .. "[91m●" .. COLOR_RESET
```

Comment:
Please add local `COLOR_` var or comment about `\[92m` meaning.

Reply:
Haha, do you know start to realize all futility of a life? ;)

You ask to "uncolor" variables, then suddenly that change makes code less readable. Who could know it?

Reply:
The code just was less readable in a different place. Thank you for fixing.